### PR TITLE
CRW-616 update to latest Che 7.8...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-616 update to latest Che 7.8 crds/roles/etc.

Change-Id: Ie777a543549f792c77b3c7580a369a63922694c6
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

File: deploy/crds/org_v1_che_cr.yaml [hunk 1]
```diff
@@ -105,7 +112,21 @@ spec:
     oAuthSecret: ''
     # image:tag used in Keycloak deployment
     identityProviderImage: ''
-
-
-
-
+  k8s:
```
Comment (davidfestal) [inline deploy/crds/org_v1_che_cr.yaml]: I don't think we should add the `k8s` block here. We're OpenShift only no ?

We should add `metrics` though

File: deploy/operator.yaml [hunk 2]
```diff
@@ -24,13 +24,24 @@ spec:
     spec:
       containers:
         - name: codeready-operator
-          image: registry.redhat.io/codeready-workspaces/server-operator-rhel8:latest
+          image: registry.redhat.io/codeready-workspaces/crw-2-rhel8-operator:latest
```
Comment (davidfestal) [inline deploy/operator.yaml]: Why `crw-2` ?

File: deploy/operator-local.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#
```
Comment (davidfestal) [inline deploy/operator-local.yaml]: I don't think you need `operator-local.yaml` file, which is mainly to run the operator locally.

File: deploy/role_binding_oauth.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#
```
Comment (davidfestal) [inline deploy/role_binding_oauth.yaml]: I don't think you need this one. It's mainly useful to run the operator locally afair.

File: deploy/operator-local.yaml (hunk 3 above)
Comment (nickboldt) [reply deploy/operator-local.yaml]: ok but it's easier to keep in sync if the same files are in both repos

File: deploy/operator.yaml (hunk 2 above)
Comment (nickboldt) [reply deploy/operator.yaml]: we needed a new name in Comet for the delivery repo, and registry.redhat.io/codeready-workspaces/codeready-workspaces-rhel8-operator seemed like so much unnecessary typing. Plus lots of other products use their acronym+version in the names.

File: deploy/role_binding_oauth.yaml (hunk 4 above)
Comment (nickboldt) [reply deploy/role_binding_oauth.yaml]: cool but again it's easier to sync if we copy everything over than if I have to have rules for which files to include/exclude

File: deploy/crds/org_v1_che_cr.yaml (hunk 1 above)
Comment (nickboldt) [reply deploy/crds/org_v1_che_cr.yaml]: k8s block removed, thx.

**Pull request merged**